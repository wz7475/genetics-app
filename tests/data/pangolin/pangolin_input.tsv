Identifier	Sex	Chr	POS	Ref	Alt	FILTER	Qual	UCSC_url	VarSome_url	Franklin_url	HGMD_url	Ensembl_url	GCexpres_url	Mastermind_url	LitVar2_url	IGV_url	AD	DP	VAF	Zyg	Mult_het	Symbol	Consequence	Impact	rs_dbSNP	inheritanceOMIM	phenotypeOMIM	OMIM_url	key	HGVS	HGVS_canonical	Position	CLNSIG	CLNDN	CLNREVSTAT	CLNSIGCONF	ClinVar_url	gnomADv3_AF	gnomADv3_AF_NFE	gnomADv3_AF_max	gnomADv3_AF_max_population	gnomADv3_AC	gnomADv3_nhomalt	gnomADv3_url	gnomADv2_AF_exomes	gnomADv2_AF_NFE_exomes	gnomADv2_AC_exomes	gnomADv2_nhomalt_exomes	varSEAK_url	SpliceRegion	ada_score	rf_score	SpliceAI_pred	SpliceAI_cutoff	LoF	pLI	SIFT	SIFT4G_pred	PolyPhen	Polyphen2_HDIV_pred	MutationTaster_pred	MutationAssessor_pred	BayesDel_addAF_pred	BayesDel_noAF_pred	LIST-S2_pred	fathmm-MKL_coding_pred	fathmm-XF_coding_pred	DEOGEN2_pred	LRT_pred	M-CAP_pred	MetaLR_pred	MetaSVM_pred	PROVEAN_pred	PrimateAI_pred	CADD_phred	CADD_phred_hg19	DANN_score	REVEL_rankscore	VEST4_rankscore	MPC_rankscore	MVP_rankscore	Eigen-PC-raw_coding_rankscore	Eigen-raw_coding_rankscore	GERP++_RS_rankscore	LRT_converted_rankscore	MutPred_Top5features	MutPred_protID	MutPred_rankscore	MutPred_score	MANE_SELECT	AAchange	Feature	ENSP	VARIANT_CLASS	BIOTYPE	Interpro_domain	hg19_pos(1-based)	OMIMcytoband	clinvar_MedGen_id	clinvar_Orphanet_id	ZygNum	vcf_REF	vcf_ALT	vcf_FORMAT	vcf_SAMPLE	design	v_hom_design	v_hom_pass_design	v_het_design	v_het_pass_design	v_hem_design	v_hem_pass_design	sn_design	sn_design_pass	design_af	design_af_pass	v_hom_exome	v_hom_pass_exome	v_het_exome	v_het_pass_exome	v_hem_exome	v_hem_pass_exome	sn_exome	sn_exome_pass	exome_af	exome_af_pass	Annotation_info
RG-corriell	female	chr10	104032447	A	C	PASS	2573	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104032444-104032450	https://varsome.com/variant/hg38/rs813943	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104032447-A-C-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs813943	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs813943%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104032397-104032497&genome=hg38&name=RG-corriell_S7	63	63	1	HOM	FALSE	COL17A1	intron_variant	MODIFIER	rs813943	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104032447_A>C	NM_000494.4:c.4439-157T>G NP_000485.3:p.?	NM_000494.4:c.4439-157T>G NP_000485.3:p.?	Intron 55/55	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1278946/	0.751446	0.757026	0.867245	eas	114328	43013	https://gnomad.broadinstitute.org/variant/10-104032447-A-C?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.00|0.00|0.00|0.00|5|27|5|-16	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	A	C	GT:AD:DP:GQ:PL	1/1:0,63:63:99:2587,190,0	Roche_RASGENODERM_V2	259	259	158	158	0	0	>50	>50	0.773455377574371	0.773455377574371	1284	1284	818	818	0	0	>50	>50	0.74977856510186	0.74977856510186	AnnotationDate:2023-03-13
RG-corriell	female	chr10	104032447	A	C	PASS	2573	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104032444-104032450	https://varsome.com/variant/hg38/rs813943	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104032447-A-C-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs813943	https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLK#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs813943%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104032397-104032497&genome=hg38&name=RG-corriell_S7	63	63	1	HOM	FALSE	SLK	downstream_gene_variant	MODIFIER	rs813943	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104032447_A>C	-	-	-	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1278946/	0.751446	0.757026	0.867245	eas	114328	43013	https://gnomad.broadinstitute.org/variant/10-104032447-A-C?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.00|0.00|0.00|0.00|5|27|5|-16	FAIL	0.699	0.43	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000369755.4	-	NM_014720.4	NP_055535.2	SNV	protein_coding	-	-	10q24.3	-	-	2	A	C	GT:AD:DP:GQ:PL	1/1:0,63:63:99:2587,190,0	Roche_RASGENODERM_V2	259	259	158	158	0	0	>50	>50	0.773455377574371	0.773455377574371	1284	1284	818	818	0	0	>50	>50	0.74977856510186	0.74977856510186	ClinVar:2023-03-06
RG-corriell	female	chr10	104033415	T	G	PASS	4349	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033412-104033418	https://varsome.com/variant/hg38/rs1747675	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033415-T-G-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs1747675	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs1747675%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033365-104033465&genome=hg38&name=RG-corriell_S7	107	107	1	HOM	FALSE	COL17A1	intron_variant	MODIFIER	rs1747675	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033415_T>G	NM_000494.4:c.4157-40A>C NP_000485.3:p.?	NM_000494.4:c.4157-40A>C NP_000485.3:p.?	Intron 52/55	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1291157/	0.98682	0.999927	1	eas	150198	74132	https://gnomad.broadinstitute.org/variant/10-104033415-T-G?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.00|0.01|0.01|0.04|-45|-24|9|-39	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	T	G	GT:AD:DP:GQ:PL	1/1:0,107:107:99:4363,322,0	Roche_RASGENODERM_V2	437	437	0	0	0	0	>50	>50	1	1	2254	2254	2	2	0	0	>50	>50	0.99867139061116	0.99867139061116	OMIM:2023-03-12
RG-corriell	female	chr10	104033415	T	G	PASS	4349	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033412-104033418	https://varsome.com/variant/hg38/rs1747675	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033415-T-G-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs1747675	https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLK#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs1747675%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033365-104033465&genome=hg38&name=RG-corriell_S7	107	107	1	HOM	FALSE	SLK	downstream_gene_variant	MODIFIER	rs1747675	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033415_T>G	-	-	-	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1291157/	0.98682	0.999927	1	eas	150198	74132	https://gnomad.broadinstitute.org/variant/10-104033415-T-G?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.00|0.01|0.01|0.04|-45|-24|9|-39	FAIL	0.699	0.43	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000369755.4	-	NM_014720.4	NP_055535.2	SNV	protein_coding	-	-	10q24.3	-	-	2	T	G	GT:AD:DP:GQ:PL	1/1:0,107:107:99:4363,322,0	Roche_RASGENODERM_V2	437	437	0	0	0	0	>50	>50	1	1	2254	2254	2	2	0	0	>50	>50	0.99867139061116	0.99867139061116	Ensembl_107-GRCh38
RG-corriell	female	chr10	104033611	G	A	PASS	284	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033608-104033614	https://varsome.com/variant/hg38/rs2181833	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033611-G-A-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2181833	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	https://mastermind.genomenon.com/detail?mutation=COL17A1:R1386int	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2181833%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033561-104033661&genome=hg38&name=RG-corriell_S7	7	7	1	HOM	FALSE	COL17A1	intron_variant	MODIFIER	rs2181833	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033611_G>A	NM_000494.4:c.4157-236C>T NP_000485.3:p.?	NM_000494.4:c.4157-236C>T NP_000485.3:p.?	Intron 52/55	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1280150/	0.695219	0.800865	0.857583	eas	105708	38759	https://gnomad.broadinstitute.org/variant/10-104033611-G-A?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.07|0.00|0.00|0.00|-5|29|-5|-16	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	G	A	GT:AD:DP:GQ:PL	1/1:0,7:7:21:298,21,0	Roche_RASGENODERM_V2	147	147	48	48	0	0	>50	>50	0.391304347826087	0.391304347826087	1274	1274	592	588	0	0	>50	>50	0.695305580159433	0.694419840566873	gnomADv3.1.2
RG-corriell	female	chr10	104033611	G	A	PASS	284	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033608-104033614	https://varsome.com/variant/hg38/rs2181833	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033611-G-A-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2181833	https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLK#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2181833%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033561-104033661&genome=hg38&name=RG-corriell_S7	7	7	1	HOM	FALSE	SLK	downstream_gene_variant	MODIFIER	rs2181833	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033611_G>A	-	-	-	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1280150/	0.695219	0.800865	0.857583	eas	105708	38759	https://gnomad.broadinstitute.org/variant/10-104033611-G-A?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.07|0.00|0.00|0.00|-5|29|-5|-16	FAIL	0.699	0.43	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000369755.4	-	NM_014720.4	NP_055535.2	SNV	protein_coding	-	-	10q24.3	-	-	2	G	A	GT:AD:DP:GQ:PL	1/1:0,7:7:21:298,21,0	Roche_RASGENODERM_V2	147	147	48	48	0	0	>50	>50	0.391304347826087	0.391304347826087	1274	1274	592	588	0	0	>50	>50	0.695305580159433	0.694419840566873	gnomADv2.1.1
RG-corriell	female	chr10	104033701	C	T	PASS	171.9	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033698-104033704	https://varsome.com/variant/hg38/rs2147268	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033701-C-T-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2147268	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	https://mastermind.genomenon.com/detail?mutation=COL17A1:R1386int	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2147268%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033651-104033751&genome=hg38&name=RG-corriell_S7	5	5	1	HOM	FALSE	COL17A1	intron_variant	MODIFIER	rs2147268	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033701_C>T	NM_000494.4:c.4156+244G>A NP_000485.3:p.?	NM_000494.4:c.4156+244G>A NP_000485.3:p.?	Intron 52/55	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1276450/	0.731096	0.802301	0.850213	eas	111033	41478	https://gnomad.broadinstitute.org/variant/10-104033701-C-T?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.26|0.24|0.00|0.00|-18|-1|-18|-1	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	C	T	GT:AD:DP:GQ:PL	1/1:0,5:5:15:186,15,0	Roche_RASGENODERM_V2	56	56	29	29	0	0	>50	>50	0.161327231121281	0.161327231121281	1110	1110	498	495	0	0	>50	>50	0.601860053144376	0.601195748449956	dbNSFPv4.2a
RG-corriell	female	chr10	104033701	C	T	PASS	171.9	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104033698-104033704	https://varsome.com/variant/hg38/rs2147268	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104033701-C-T-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2147268	https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLK#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2147268%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104033651-104033751&genome=hg38&name=RG-corriell_S7	5	5	1	HOM	FALSE	SLK	downstream_gene_variant	MODIFIER	rs2147268	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104033701_C>T	-	-	-	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1276450/	0.731096	0.802301	0.850213	eas	111033	41478	https://gnomad.broadinstitute.org/variant/10-104033701-C-T?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.26|0.24|0.00|0.00|-18|-1|-18|-1	FAIL	0.699	0.43	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000369755.4	-	NM_014720.4	NP_055535.2	SNV	protein_coding	-	-	10q24.3	-	-	2	C	T	GT:AD:DP:GQ:PL	1/1:0,5:5:15:186,15,0	Roche_RASGENODERM_V2	56	56	29	29	0	0	>50	>50	0.161327231121281	0.161327231121281	1110	1110	498	495	0	0	>50	>50	0.601860053144376	0.601195748449956	DesignSamples:437
RG-corriell	female	chr10	104036641	G	C	PASS	4799	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104036638-104036644	https://varsome.com/variant/hg38/rs2274098	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104036641-G-C-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2274098	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	https://mastermind.genomenon.com/detail?mutation=COL17A1:R1093int	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2274098%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104036591-104036691&genome=hg38&name=RG-corriell_S7	117	117	1	HOM	FALSE	COL17A1	splice_polypyrimidine_tract_variant,intron_variant	LOW	rs2274098	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104036641_G>C	NM_000494.4:c.3278-9C>G NP_000485.3:p.?	NM_000494.4:c.3278-9C>G NP_000485.3:p.?	Intron 47/55	Benign	Epithelial_recurrent_erosion_dystrophy|Junctional_epidermolysis_bullosa,_non-Herlitz_type|not_specified|not_provided	criteria_provided,_multiple_submitters,_no_conflicts	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/256274/	0.730251	0.833127	0.840147	eas	110985	42104	https://gnomad.broadinstitute.org/variant/10-104036641-G-C?dataset=gnomad_r3	0	0	0	0	https://varseak.bio/ssp.php?gene=COL17A1&transcript=NM_000494.4&variant=&hgvs=c.3278-9C>G	splice_polypyrimidine_tract_variant	0.00320531925180129	0.056	COL17A1|0.04|0.01|0.00|0.00|10|-9|25|-12	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	G	C	GT:AD:DP:GQ:PL	1/1:0,117:117:99:4813,352,0	Roche_RASGENODERM_V2	299	299	129	129	0	0	>50	>50	0.831807780320366	0.831807780320366	1515	1515	650	650	0	0	>50	>50	0.814880425155005	0.814880425155005	ExomeSamples:2258
RG-corriell	female	chr10	104036691	A	G	PASS	3592	http://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=chr10%3A104036688-104036694	https://varsome.com/variant/hg38/rs2476957	https://franklin.genoox.com/clinical-db/variant/snp/chr10-104036691-A-G-hg38	-	http://www.ensembl.org/homo_sapiens/Variation/Summary?v=rs2476957	https://www.genecards.org/cgi-bin/carddisp.pl?gene=COL17A1#expression	-	https://www.ncbi.nlm.nih.gov/research/litvar2/docsum?variant=litvar@rs2476957%23%23	http://localhost:60151/load?file=http://10.1.23.124:1337/Roche_RASGENODERM_V2/panel_rasgenoderm/run_25/panel_rasgenoderm-run_25-Apr_11_2020/final/RG-corriell_S7/RG-corriell_S7-ready.bam&locus=10:104036641-104036741&genome=hg38&name=RG-corriell_S7	88	88	1	HOM	FALSE	COL17A1	intron_variant	MODIFIER	rs2476957	AD,AR	Epidermolysis bullosa, junctional 4, intermediate, 619787 (3),Epithelial recurrent erosion dystrophy, 122400 (3)	https://www.omim.org/entry/113811	chr10:104036691_A>G	NM_000494.4:c.3278-59T>C NP_000485.3:p.?	NM_000494.4:c.3278-59T>C NP_000485.3:p.?	Intron 47/55	Benign	not_provided	criteria_provided,_single_submitter	-	https://www.ncbi.nlm.nih.gov/clinvar/variation/1291156/	0.987898	0.999897	1	eas	150277	74247	https://gnomad.broadinstitute.org/variant/10-104036691-A-G?dataset=gnomad_r3	0	0	0	0	-	-	-	-	COL17A1|0.00|0.01|0.00|0.00|22|-40|50|33	FAIL	0.104	0.00	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	ENST00000648076.2	-	NM_000494.4	NP_000485.3	SNV	protein_coding	-	-	10q24.3	-	-	2	A	G	GT:AD:DP:GQ:PL	1/1:0,88:88:99:3606,264,0	Roche_RASGENODERM_V2	437	437	0	0	0	0	>50	>50	1	1	2255	2255	2	2	0	0	>50	>50	0.99911426040744	0.99911426040744	""
