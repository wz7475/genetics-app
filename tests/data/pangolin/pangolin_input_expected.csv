CHROM,POS,REF,ALT
10,104032447,A,C
10,104032447,A,C
10,104033415,T,G
10,104033415,T,G
10,104033611,G,A
10,104033611,G,A
10,104033701,C,T
10,104033701,C,T
10,104036641,G,C
10,104036691,A,G
